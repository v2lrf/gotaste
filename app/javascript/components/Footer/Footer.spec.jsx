import React from 'react'
import { Footer } from '.'

describe('Footer', () => {
  let component
  beforeEach(() => {
    component = shallow(<Footer />)
  })

  describe('Snapshots', () => {
    it('should match the snapshot', () => {
      expect(component).toMatchSnapshot()
    })
  })
})
