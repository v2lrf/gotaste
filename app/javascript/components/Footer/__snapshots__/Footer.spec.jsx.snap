// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snapshots should match the snapshot 1`] = `
<Fragment>
  <Container>
    <div
      className="flex border-b py-10"
    >
      <div
        className="pr-16"
      >
        <div
          className="pb-2"
        >
          <a
            className="text-black no-underline"
            href="#forhandlere"
          >
            Forhandlere
          </a>
        </div>
        <div>
          <a
            className="text-black no-underline"
            href="#arrangementer"
          >
            Arrangementer
          </a>
        </div>
      </div>
      <div
        className="pr-16"
      >
        <div
          className="pb-2"
        >
          <a
            className="text-black no-underline"
            href="#om-os"
          >
            Om Govinu
          </a>
        </div>
        <div>
          <a
            className="text-black no-underline"
            href="#kontakt"
          >
            Kontakt
          </a>
        </div>
      </div>
      <div>
        <div
          className="pb-2"
        >
          <a
            className="text-black no-underline"
            href="#betingelser"
          >
            Betingelser
          </a>
        </div>
        <div
          className="pb-2"
        >
          <a
            className="text-black no-underline"
            href="#vilkaer"
          >
            PrivatlivsvilkÃ¥r
          </a>
        </div>
        <div>
          <a
            className="text-black no-underline"
            href="#cookie"
          >
            Cookiepolitik
          </a>
        </div>
      </div>
    </div>
    <div
      className="flex justify-between py-10"
    >
      <div>
        Â© 2019 Copyright Govinu. All rights reserved.
      </div>
      <div>
        <span
          aria-label="Dansk"
          role="img"
        >
          ðŸ‡©ðŸ‡°
        </span>
        Dansk
      </div>
    </div>
  </Container>
</Fragment>
`;
